test/jdk/ProblemList-Xcomp.txt

```diff
@@ -29,3 +29,6 @@
 
 java/lang/invoke/MethodHandles/CatchExceptionTest.java 8146623 generic-all
 java/lang/ref/ReferenceEnqueue.java 8284236 generic-all
+
+com/sun/jdi/JdbLastErrorTest.java 8293829 generic-all
```

Comment:
Should this be windows-x64 instead of generic-all?

Reply:
Yes, updating that.  I  was thinking it might save some time somewhere to make it generic, but that's probably overthinking.